src/data/selectors/special.test.js

```diff
@@ -0,0 +1,27 @@
+import selectors from './special';
+
+describe('hasSpecialBulkManagementAccess', () => {
+  // Copy & restore process for testing purposes
+  const OLD_ENV = process.env;
+  const allowedCourses = 'edX/DemoX/2021T1,edX/DemoX/2021T2';
+
+  beforeEach(() => {
+    jest.resetModules();
```

Comment:
what is the purpose of this reset?  I've only used this when programatically importing.